package.json

```diff
@@ -6,13 +6,9 @@
     "nodeshift": "./bin/nodeshift"
   },
   "scripts": {
-    "lint": "eslint bin/* lib/**/*.js test/*-test.js test/**/*-test.js index.js",
-    "test": "cross-env NODESHIFT_QUIET=true tape test/*-test.js test/**/*-test.js | tap-spec",
-    "coverage": "nyc npm test",
-    "coverage:html": "nyc npm test && nyc report --reporter=html",
```

Comment:
can we keep this one.  I find it useful when i run things locally to see where i need coverage

Reply:
when we run `npm it` we already have lint + test + coverage

Reply:
yea,  i know,  but `coverage:html` outputs a nice html version that i look at when doing local development

Reply:
coverage html report added back :+1:

Reply:
`npm run coverage`